core/src/main/resources/hudson/PluginManager/available_pl.properties

```diff
@@ -20,11 +20,14 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 Available\ Plugins=Dostępne wtyczki
```

Comment:
`core/src/main/resources/hudson/PluginManager/available.jelly` no longer contains an "Available Plugins" string with a capital P, so this line can be deleted:

```suggestion
```